processor/tasks/helpers/helpers.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+
+def convert_to_comma_seperated(list):
```

Comment:
Perhaps, this argument should, nevertheless the scoped usage, not be named as `list`?

Reply:
changed